packages/shared/src/store/groupActions.ts

```diff
@@ -50,6 +50,17 @@ export async function createGroup({
   }
 }
 
+export async function getGroupPreview(groupId: string) {
```

Comment:
Seems largely the same as `sync.syncGroupPreviews`, can we combine the two? I'm agnostic as to which we keep.